assets/questions/mostCommonChar.js

```diff
@@ -56,19 +76,30 @@ globalData.questions['common'] = {  // eslint-disable-line dot-notation
       input: 'apoiuytrewqsdf*&^%$#ba',
       allowedOutputs: ['a']
     }, {
-      input: 'abb4562873ba',
-      allowedOutputs: ['b']
+      input: 'aBBB4562873ba',
+      allowedOutputs: ['B']
     }, {
       input: 'babaabb',
       allowedOutputs: ['b']
     }],
-    buggyOutputTests: [],
+    buggyOutputTests: [{
```

Comment:
These don't seem like bugs -- it seems unlikely that a student would manage to write a function for this that only checks for lowercase characters or alphabetic characters. Plus, your correctness tests already capture those. Can you think of a bug that might be a bit less common, but is still possible?